game_event/tests.py

```diff
@@ -265,3 +296,53 @@ def test_fail_add_player_to_event_already_in_it(self, client, saved_game_event,
         assert response.status_code == 200
         with pytest.raises(IntegrityError):
             response = client.post(url, {'answer': 'yes'})
+
+
+@pytest.mark.django_db
+class TestWeatherApi:
```

Comment:
We're testing our function rather than the API.
```suggestion
class TestGetWeatherFromApi:
```

Reply:
Changed